tests/test_aggregation.py

```diff
@@ -72,15 +82,20 @@
 class AggeregateEngineTests(IsolatedAsyncioTestCase):
   def _equality_check(self, aidb_res, gt_res, error_target):
     assert len(aidb_res) == len(gt_res)
+    error_rate_list = []
+    valid_estimation = True
     for aidb_item, gt_item in zip(aidb_res, gt_res):
-      if abs(aidb_item - gt_item) / (gt_item) <= error_target:
-        return True
-    return False
+      relative_diff = abs(aidb_item - gt_item) / (gt_item)
+      error_rate_list.append(relative_diff * 100)
+      if relative_diff > error_target:
+        valid_estimation = False
+    logger.info(f'Error rate (%) for approximate aggregation query: {error_rate_list}')
+    return valid_estimation
 
 
   async def test_agg_query(self):
     count_list = [0] * len(queries)
-    for i in range(100):
+    for i in range(10):

```

Comment:
don't modify this since it isn't run on github until we fix the number of trials